pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>io.kokuwa.maven</groupId>
+		<artifactId>maven-parent</artifactId>
+		<version>0.6.13</version>
+		<relativePath />
+	</parent>
+
+	<groupId>com.boozallen.aissemble</groupId>
+	<artifactId>helm-dependency-maven-plugin</artifactId>
+	<version>6.13.0-SNAPSHOT</version>
+	<packaging>maven-plugin</packaging>
+
+	<name>Helm Dependency Maven Plugin</name>
+	<description>A plugin for deploying Helm dependencies to a remote repository.</description>
+	<url>https://github.com/TechnologyBrewery/helm-dependency-maven-plugin</url>
+
+	<properties>
+
+		<!-- ===================================================================== -->
+		<!-- ============================== Build ================================ -->
+		<!-- ===================================================================== -->
+
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<maven.compiler.showDeprecation>false</maven.compiler.showDeprecation>
+		<maven.compiler.compilerArgument>-Xlint:all,-processing,-deprecation</maven.compiler.compilerArgument>
+
+		<!-- we have flaky tests with caching of Authenticator.setDefault() -->
+		<surefire.rerunFailingTestsCount>3</surefire.rerunFailingTestsCount>
+
+		<invoker.debug>false</invoker.debug>
+		<invoker.parallelThreads>0.5C</invoker.parallelThreads>
+		<invoker.settingsFile>${project.basedir}/src/it/settings.xml</invoker.settingsFile>
+
+		<!-- ===================================================================== -->
+		<!-- ============================= Versions ============================== -->
+		<!-- ===================================================================== -->
+
+		<version.com.puppycrawl.tools.checkstyle>9.3</version.com.puppycrawl.tools.checkstyle>
+        <group.helm.maven.plugin>io.kokuwa.maven</group.helm.maven.plugin>
```

Comment:
Q/S: Do you not need to use the tech brewery version for now, since it has the extensions updates?

Reply:
The forked tech brewery version actually does not change the group ID.

Reply:
Ah, right.  If we want to get this released without waiting on the upstream PR approval, we'll want to change the group and do a release under the tech brewery group then change this to tech brewery.  All depends on what our needs are. (We've done this pattern of releasing the changes early under tech brewery a few times now so there are examples of the process if we need to do it.)